mobile_app/README.md

```diff
@@ -0,0 +1,30 @@
+Mobile Apps
+===========
+
+Tools contained here are used for working with the mobile-apps
+
+trigger_build
+-------------
+This will:
+
+* Write a known set of environment variables to disk in JSON format to a file
+  named CONFIGURATION.
```

Comment:
It's saved to the jenkins workspace and then pushed. ``master`` on the repo itself is essentially empty so I think the name should be fine.